### PR TITLE
Deprecate old endpoint. Add new one

### PR DESCRIPTION
**Pull request by LuisOw**

### CODE REVIEW

**Comment by vitorezende** [comment]
We can change all tests from /operations endpoint to test /files endpoint and mark endpoints /operations with @Deprecated annotation

**Review by vitorezende — APPROVED** [review]
Great!

File: docs/field_boundaries_endpoints.md [hunk 1]
```diff
@@ -561,7 +565,109 @@ They are:
 ]
 ```
 
-### Get an operation file of a field
+### Get all operation files of a field
+
+&nbsp<span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/files`
```
Comment (vcruvinelr) [inline docs/field_boundaries_endpoints.md:570]: @LuisOw for this endpoints maybe we can update to `/users/{leafUserId}/....?

File: docs/field_boundaries_endpoints.md [hunk 2]
```diff
@@ -2165,8 +2352,10 @@ Each boundary has a `status` and `providerStatus`.
 
 | Description | Endpoints
 | - | - |
-[Get all operations of a field](#get-all-operations-of-a-field) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations`
-[Get an operation of a field](#get-an-operation-of-a-field) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/{id}`
+[Get all operations of a field (deprecated)](#get-all-operation-files-of-a-field-deprecated) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations`
+[Get all operations of a field](#get-all-operation-files-of-a-field) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/files`
+[Get an operation of a field (deprecated)](#get-an-operation-of-a-field-deprecated) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/{id}`
+[Get an operation of a field](#get-an-operation-of-a-field) | <span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/files/{id}`
```
Comment (vcruvinelr) [inline docs/field_boundaries_endpoints.md:2358]: Same here.

**Review by vcruvinelr — APPROVED** [review]

File: docs/field_boundaries_endpoints.md (hunk 1 above)
Comment (LuisOw) [reply docs/field_boundaries_endpoints.md]: 5617309 Done

File: docs/field_boundaries_endpoints.md [hunk 3]
```diff
@@ -618,6 +724,87 @@ Gets a single Operation File of a field by its id.
...
+    "varieties": [
+      "string"
+    ]
+  }
+]
+```
+
+### Get an operation file of a field 
+
+&nbsp<span class="badge badge--success">GET</span> `/users/{id}/fields/{id}/operations/files/{id}`
```
[the remark this answers is not in the document]
Comment (LuisOw) [reply docs/field_boundaries_endpoints.md]: 5617309 Done

File: docs/field_boundaries_endpoints.md (hunk 2 above)
Comment (LuisOw) [reply docs/field_boundaries_endpoints.md]: 5617309 Done

**Pull request merged**